docs/guides/formulasC2/examples.md

```diff
@@ -0,0 +1,275 @@
+  10. In **API**, enter the endpoint to make a bulk query.
+
+            /hubs/crm/bulk/query
+
+  11. Click **Show Advanced**.
+  12. In **Headers**, enter the reference to the headers that you built in the script in the `buildMetaData` step. In this case, type `${steps.buildMetaData.headers}`.
+  13. In **Query**, enter the reference to the query that you built in the script in the `buildMetaData` step. In this case, type `${steps.buildMetaData.query}`.
+  13. Click **Save**.
+
+The first formula is finished and should look like the visualization below. It should include a trigger and two steps: the first builds the metadata for a bulk query, and the second makes the bulk query, which includes a callback to the formula execution endpoint of the next formula.
+![Trigger](img/viz-bulk-transfer-1.png)
+
+#### Formula 2
+
+To create a formula that receives the notification that the job is done, downloads the file from the original element and posts to the destination:
+
+1. [Build a formula template](#build-a-formula-template) and select **Manual** as the trigger, and then click **Save**.
+
+    {% include note.html content="You do not need to configure anything for the manual trigger, but take note of the endpoint that you will need to trigger the formula: `POST /formulas/instances/:id/executions`" %}
+
+4. Add two element instance variables to represent the element that you are downloading from and the element that you are uploading to, and a variable to represent the resource that you are syncing.
+  5. Click **Variables**.
+  ![Variables](img/variables.png)
+  7. Click **Value**.
+  8. Enter a name for variable that represents the resource that you want to sync. For this tutorial we'll call it `resourceName`.
+  9. Click **Save**.
+  11. Create Element Instance variables to represent the source and target systems to sync. For this example, use `originInstance` and `destinationInstance`.
+10. In the formula visualization, click <img src="img/btn-add-step.gif" alt="Add a Step" class="inlineImage"> to add a step.
+6. Create a JS Filter step that makes sure that the bulk query is completed.
+  7. Click **JS Filter (true/false)**.
+  8. Enter a name for the script. We'll call it `isSuccessful`.
+  8. Enter a script such as the example below.
+
+      ```js
+      let status = trigger.args.status;
+
+      if (status && status === "COMPLETED") {
+        done(true);
+      } else {
+        done(false);
+      }
+      ```
+6. Create a JS Script step that defines an identifier field, which is the unique key for an upsert operation. It also specifices the content type as **csv**. Under the **isSuccessful** step, click the **OnSuccess** button  <img src="img/btn-onSuccess.png" alt="OnSuccess" class="inlineImage">.
+  7. Click **JS Script**.
+  8. Enter a name for the script. We'll call it `buildMetaData`.
+  8. Enter a script like the following example:
+
+      ```js
+      let metaData = {
```

Comment:
A @jjwyse Street cred is of the utmost importance, but I have none. So, instead of 'let metaData = ...` I should change to 'const metaData = ...`. Correct?